games-simulation/thepowdertoy/thepowdertoy-91.5.330.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="The Powder Toy - A free physics sandbox game"
+DESCRIPTION="The Powder Toy is a free physics sandbox game,\
+which simulates air pressure and velocity, heat, gravity and \
+a countless number of interactions between different substances! \
+The game provides you with various building materials, liquids, gases \
+and electronic components which can be used to construct complex machines,\
+guns, bombs, realistic terrains and almost anything else."
+HOMEPAGE="http://powdertoy.co.uk"
+COPYRIGHT="2008 - 2010 Stanislaw Skowronek.
+	2010 Simon Robertshaw"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/simtr/The-Powder-Toy/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="9a7de557e771f88e823d06cc21d16f1fbbca930ce751720bc9035d00073d6635"
+SOURCE_DIR="The-Powder-Toy-$portVersion"
+PATCHES="thepowdertoy-$portVersion.patchset"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	powder$secondaryArchSuffix = $portVersion compat >= 1
+	cmd:powder$secondaryArchSuffix = 1.0.0 compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+	lib:libfftw3$secondaryArchSuffix
+	lib:libfftw3f$secondaryArchSuffix
+	lib:liblua$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl$secondaryArchSuffix_devel
+	lua51${secondaryArchSuffix}_devel
+	devel:liblua$secondaryArchSuffix_devel
+	devel:liblua5.1$secondaryArchSuffix_devel
+	fftw$secondaryArchSuffix_devel
+	devel:libfftw3$secondaryArchSuffix_devel
+	devel:libfftw3f$secondaryArchSuffix_devel
+	devel:libbz2$secondaryArchSuffix_devel
+	devel:libz$secondaryArchSuffix_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:cmake
+	cmd:python
+	cmd:scons
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	scons -Q
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	cp build/powder $appsDir/
```

Comment:
I'd prefer to have the app name capitalized, so "Powder", as practically all GUI apps in Haiku are.